css/src/components/headings.css

```diff
@@ -33,6 +33,7 @@ body.yoast {
 
 .yoast h1, .yoast .h1 {
 	font-size: 1.7em;
+	margin-bottom: 1.34em;
```

Comment:
On a general note: please maybe consider values expressed in relative units are then computed by browsers to pixel values. Browsers have rounding mechanisms that differ. When using relative units, ideally we should make sure the computed values translate to _pixels without decimal values_. A pixel on the screen can't be split, not yet at least. It's very likely these values will trigger browsers rounding with different results on different browsers.

Actually, these values translate to:
```
font-size: 23.8px;
line-height: 28.56px;
```

I do realize this is not specific to this PR and should be considered at a higher level.